firebase.go

```diff
@@ -65,6 +68,20 @@ func sanitizeURL(url string) string {
 	return url
 }
 
+func sanitizePath(p string) string {
+	// remove slashes from the front and back
+	//	/foo/.json -> foo/.json
+	s := strings.Trim(p, "/")
```

Comment:
`p` is passed by value. can't we reuse `p` throughout this func instead of allocating `s`?